tensorbay/client/version.py

```diff
@@ -638,9 +641,12 @@ def _generate_jobs(
     ) -> Generator[SquashAndMergeJob, None, int]:
         response = self._list_jobs(self._JOB_TYPE, status, offset, limit)
         for item in response["jobs"]:
-            yield SquashAndMergeJob.from_response_body(
+            job = SquashAndMergeJob.from_response_body(
                 item, dataset_id=self._dataset_id, client=self._client, job_updater=self._get_job
             )
+            job._draft_getter = self._draft_getter  # pylint: disable=protected-access
```

Comment:
shall we put the assignment of `_draft_getter` into `SquashAndMergeJob.from_response_body`